### DISCUSSION TITLE
Is it possible to stop the deployment email notifications?

### DISCUSSION THREAD

**Discussion by jamesmacwhite** [craftcms/cloud#41] [Q&A] [answered]
I can't see any way to stop the email notifications related to deployments that have been pushed to the branch an environment is looking at.

Is there some form of notification settings to change?

**Comment by angrybrad** [comment, marked as answer]
Not currently, but there will be once we implement https://github.com/craftcms/cloud/discussions/27

**Reply by jamesmacwhite** [reply]
Ah OK, thanks for highlighting the existing feature request though. Definitely would be useful, as currently our owner account which is registered under a general mailbox is being spammed by the deployment notifications!

I can implement a mail rule to automatically archive them for now.

**Reply by timkelty** [reply]
@jamesmacwhite part of the challenge is to make them customizable without making the management a pain. Which ones are you finding you want to mute the most?

**Reply by jamesmacwhite** [reply]
> @jamesmacwhite part of the challenge is to make them customizable without making the management a pain. Which ones are you finding you want to mute the most?

I wouldn't say a pain, just not required in my case, given the email address for the owner of the Craft Console account is a general mailbox, so it is not constantly monitored and just adding noise. I typically know when I'm pushing code, so I don't need to be notified really. I can understand the need for larger teams/multiple developers though.

I feel there's a balance of allowing some control of transactional messages or allow it to be an opt-in/out.
